### ISSUE TITLE
Allow option to use friendly name of account for imports

### ISSUE THREAD

**Issue by wallyqs** [nats-io/nsc#241]
Currently need to use the public nkey in order to add an import, so have to shell out to `nsc` again:

```sh
        # Create system account
        nsc add account --name SYS
        nsc add user    --name sys

        # Create a couple of accounts (A & B) for testing purposes.
        nsc add account --name A
        nsc add user -a A \
                     --name test \
                     --allow-pubsub 'test' \
                     --allow-sub latency.on.test

        # Add latency exporting for the test subject from account A.
        nsc add export  -a A  --latency latency.on.test --sampling 100 --service -s test

        # Add account B that imports services from A.
        nsc add account --name B
        nsc add user -a B \
                     --name test \
                     --allow-pubsub 'test'

        nsc add import --account B \
                       --src-account $(nsc list accounts 2>&1 | awk '$2 == "A" {print $0}' | awk '{print $4}') \
                       --remote-subject test --service --local-subject test
```

**Comment by aricart** [comment]
If you control both accounts, you can also add the import interactively and not worry about any of those flags. Let me see what I can do about the names, should be able to repurpose the flag to to match an account name, that would make it much easier.

**Comment by tbeets** [comment]
+1 for friendly source account name for non-private imports.   Probably need both a ref (pub key) option and friendly name option to handle different deploy scenarios.  Friendly name option would essentially be a convenience local lookup as @wallyqs notes.

**Comment by matthiashanel** [comment]
This has been fixed by #396
